### PR TITLE
issue_2475_roman_numerals_max_3999

### PR DESCRIPTION
**Pull request by dbh**
Adjust Instructions and Unit Tests to support maximum Roman numeral 3999
Issue https://github.com/exercism/go/issues/2475

Adjustment to problem specification, instructions, and unit tests to support the max val 3,999 which can be expressed in Roman numerals (without getting into the "Larger number" scenarios)

### CODE REVIEW

**Comment by dbh** [comment]
Issue write-up is at https://github.com/exercism/problem-specifications/issues/2095 
It is more appropriate in project specifications than in the go language project since the change in spec instructions and unit tests affects more than go.

**Review by jiegillet — APPROVED** [review]
Sounds good to me, I wouldn't expect tracks implemented an artificial cutoff at 3000 (although my expectations aren't worth much)

**Comment by axtens** [comment]
Suggestion: have another set of tests that require the implementation of the [Unicode extensions](https://en.wikipedia.org/wiki/Roman_numerals#Unicode) to Roman numerals

**Comment by IsaacG** [comment]
@junedev More reviews are needed here. Mind adding one?

**Comment by glennj** [comment]
> Suggestion: have another set of tests that require the implementation of the [Unicode extensions](https://en.wikipedia.org/wiki/Roman_numerals#Unicode) to Roman numerals

Sounds like that would be a brand new exercise (like resistor-color is a series of exercises)

**Comment by dbh** [comment]
> > Suggestion: have another set of tests that require the implementation of the [Unicode extensions](https://en.wikipedia.org/wiki/Roman_numerals#Unicode) to Roman numerals
> 
> 
> 
> Sounds like that would be a brand new exercise (like resistor-color is a series of exercises)

Agreed. I think Unicode Roman numerals would be a new effort and problem spec. 

This particular PR is to clarify an existing problem spec.

File: exercises/roman-numerals/description.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ into stone tablets).
  7  => VII
 ```
 
-There is no need to be able to convert numbers larger than about 3000.
+The maximum number supported by this this notation is 3,999.
```
Comment (petertseng) [inline exercises/roman-numerals/description.md:23]:
```suggestion
The maximum number supported by this notation is 3,999.
```

**Review by junedev — APPROVED** [review]

**Review by IsaacG — APPROVED** [review]
